site/static/detailed-query.html

```diff
@@ -52,17 +52,17 @@
         <a href="dashboard.html">dashboard</a>, <a href="status.html">status</a>.</div>
     <div id="content">
         <h3 id="title"></h3>
-        <p>Totals row '%total time' is as a comparison to the 'cpu-clock' stat (we do not use
+        <p>'Time (%)' is the percentage of the cpu-clock time spent on this query.
+        Totals row is as a comparison to the 'cpu-clock' stat (we do not use
         wall-time as we want to account for parallelism).</p>
-        <p>% total time is the percentage of the cpu-clock time spent on this query.</p>
```

Comment:
This text sounds rather redundant now, but I just reordered and merged the previous two paragraphs about the sae column. To me it sounds like it is saying twice that time is relative to 'cpu-clock'? Or is "the cpu-clock time" and "the 'cpu-clock' stat" not the same thing? It is somewhat confusing.

Reply:
Hm, yes. Those are the same thing and especially now that you've merged the two sentences it definitely sounds repeated.

I had intended to have two because one references the value in the header and the other the values in the column, but they're all percentages of the CPU clock, so should be fine to merge them.

Reply:
Done.